jwst/spectral_leak/spectral_leak_step.py

```diff
@@ -12,102 +12,112 @@
 
 class SpectralLeakStep(Step):
     """
+    Apply a spectral leak correction to the Channel 3A of MIRI MRS data.
+
     The MIRI MRS has a spectral leak in which 6 micron light leaks into the
     12 micron channel.  This step applies a correction to the 12 micron channel.
     """
 
     class_alias = "spectral_leak"
 
-    reference_file_types = ['mrsptcorr']
+    reference_file_types = ["mrsptcorr"]
 
-    def process(self, input):
-        """Execute the step.
+    def process(self, input_data):
+        """
+        Apply a spectral leak  correction to MIRI MRS Channel 3A data.
 
         Parameters
         ----------
-        input : container of models containing 1-D extracted spectra
+        input_data : ~jwst.datamodels.ModelContainer
+            Container of models containing 1-D extracted spectra
 
         Returns
         -------
         JWST DataModel
-            The corrected data model. This will be "input" if the step is skipped,
+            The corrected data model. This will be the input_model if the step is skipped,
```

Comment:
Should be "input model" (with a space, not an underscore) here.
```suggestion
            The corrected data model. This will be the input model if the step is skipped,
```